app/liquid_tags/jsitor_tag.rb

```diff
@@ -1,7 +1,7 @@
 class JsitorTag < LiquidTagBase
   PARTIAL = "liquids/jsitor".freeze
   URL_REGEXP = /\A(https|http):\/\/jsitor\.com\/embed\/\w+[?a-zA-Z&]*\Z/.freeze
```

Comment:
Tihs doesn't work with and ID with the dash, like `https://jsitor.com/embed/2o-syYxmi`. Don't forget to add a test for it!

Reply:
should have read last pr #4762 and understand the code more. sorry about that. gonna add it now